### PR TITLE
Bug 1987152: Deploy CNV for testing update

### PR DESCRIPTION
**Pull request by yaacov**
Update the CNV deployment before tests to new versions

### CODE REVIEW

**Comment by yaacov** [comment]
@metalice @avivtur @pcbailey @glekner please review

File: frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh [hunk 1]
```diff
@@ -2,7 +2,73 @@
...
+  'windows10-desktop-large'
+)
+
+ANNOTATIONS=(
+  template.kubevirt.io/provider='Red Hat'
+  template.kubevirt.io/provider-url='https://www.redhat.com'
+  template.kubevirt.io/provider-support-level=Full
+)
+
+# Wait until master and worker MCP are Updated
```
Comment (pcbailey) [inline frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh:46]: nitpick: "Updated" shouldn't be capitalized here or in the messages below.

File: frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh [hunk 2]
```diff
@@ -54,35 +120,31 @@ oc create -f https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-o
...
 # Create storage class and storage namespace for testing
+# Install HPP
 
-# Create openshift-cnv namespace for Integration Tests
-oc create namespace openshift-cnv
+export HPP_VERSION="release-v0.8"
 
-# Setup storage
-oc create -f https://raw.githubusercontent.com/openshift/console/master/frontend/packages/kubevirt-plugin/integration-tests/ci-scripts/hostpath-provisioner-setup.yml
+# Configure SELinux when using OpenShift, will allow hpp to create storage on workers.
```
Comment (pcbailey) [inline frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh:129]: nitpick: Change to "Configure SELinux when using OpenShift to allow HPP to create storage on workers"

File: frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh [hunk 3]
```diff
@@ -186,3 +186,8 @@ done
 
 # Unpause the SSP operator
 oc annotate -n kubevirt-hyperconverged deployments ssp-operator kubevirt.io/operator.paused-
+
+# ----------------------------------------------------------------------------------------------------
+# Create openshift-cnv namespace for Integration Tests
```
Comment (pcbailey) [inline frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh:191]: nitpick: "Integration Tests" => "integration tests"

File: frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh [hunk 4]
```diff
@@ -2,7 +2,73 @@
...
+    local mcp_updated="false"
+
+    sleep 30
+
+    for i in {1..60}
+    do
+      echo "Attempt ${i}/60"
+      sleep 30
+      if oc wait mcp master worker --for condition=updated --timeout=1m; then
+        echo "MCP is Updated"
```
Comment (pcbailey) [inline frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh:59]: "Updated" => "updated"

File: frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh [hunk 5]
```diff
@@ -2,7 +2,73 @@
...
+        mcp_updated="true"
+        break
+      fi
+    done
+
+    if [[ "$mcp_updated" == "false" ]]; then
+      echo "Error: MCP didn't get Updated!!"
```
Comment (pcbailey) [inline frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh:66]: "Updated" => "updated"

**Pull request merged**